🎉  Add end of project event

SaranjeetKaur opened the pull request:
This PR adds details of the end of project event scheduled for 27th March, 2025.

To-do:
- [x] Find and add an image
- [ ] Fix registration form to show all questions instead of only the first 5

SaranjeetKaur commented:
@eirini-zormpa please review whenever you can.

jcohen02 commented on _pages/events/2025-03-27_project-end.md:
```diff
@@ -0,0 +1,64 @@
+
+## Overview
+
+- 🕒 **Thursday 27th February** 2025, **[10:00 - 16:00 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=HPC+Ready+-+Three+years+of+developing+RSE+skills+with+UNIVERSE-HPC+%28tentative%29&iso=20250327T10&p1=1440&ah=6)**
```
I think this should say March 🙂

jcohen02 commented on collections/_events/20250327-project-end.md:
```diff
@@ -0,0 +1,14 @@
+event_page: 2025-03-27_project-end
+slides: ""
+video: ""
+overview: "This is the UNIVERSE-HPC end of project celebration event. This will be a great opportunity to network with people involved with the project and also the wider RSE and HPC community!<br/><a href=\"https://www.universe-hpc.ac.uk/events/code-of-conduct\" target=\"_blank\">Event code of conduct</a>"
```
I'm suggesting a possible rewording here so that it's clear that this is not just a celebration of the project and what we've been doing but that it's also a wider community event with talks from other projects. See what you think and feel free to edit or reject this suggestion.
```suggestion
overview: "This event will mark the end of the UNIVERSE-HPC project. We'll have talks from members of the project as well as the wider RSE and HPC training community. This will be a great opportunity to meet and network with members of the community and find out more about the work going on in this area!<br/><a href=\"https://www.universe-hpc.ac.uk/events/code-of-conduct\" target=\"_blank\">Event code of conduct</a>"
```

SaranjeetKaur commented:
@eirini-zormpa I've updated the title, added some more content and an image. 

The registration form still shows 5 questions in the preview instead of 8 questions and I can't seem to figure out what is happening. Any idea?

eirini-zormpa commented:
Awesome, thank you Saranjeet 😊 I'll have a look tomorrow if that's okay!

eirini-zormpa commented on _pages/events/2025-03-27_project-end.md:
```diff
@@ -0,0 +1,62 @@
+
+Everyone is welcome!
+
+### [Register now (form to be updated)](https://forms.office.com/Pages/ResponsePage.aspx?id=B3WJK4zudUWDC0-CZ8PTB5xW6C9sTFJAj9JEr0thFURUOTlJQ1RDUDlZUkxNTDJBMVdBSzRDOUlDQi4u)
```
```suggestion
### [Register now](https://forms.office.com/Pages/ResponsePage.aspx?id=B3WJK4zudUWDC0-CZ8PTB5xW6C9sTFJAj9JEr0thFURUOTlJQ1RDUDlZUkxNTDJBMVdBSzRDOUlDQi4u)
```

jcohen02 replied:
You could also potentially use the short link for the form here, although I guess it doesn't actually matter?! 🙂

eirini-zormpa commented on _pages/events/2025-03-27_project-end.md:
```diff
@@ -0,0 +1,62 @@
+## Overview
+
+- 🕒 **Thursday 27th March** 2025, **[10:00 - 16:00 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=HPC+Ready%3A+Developing+RSE+and+HPC+skills&iso=20250327T10&p1=1440&ah=6)**
+- 🌍 In person at the University of Edinburgh, Bayes Centre and Online via Zoom
```
```suggestion
- 🌍 **In person** at the University of Edinburgh, Bayes Centre and **Online** via Zoom
```

eirini-zormpa commented on _pages/events/2025-03-27_project-end.md:
```diff
@@ -0,0 +1,62 @@
+- 🕒 **Thursday 27th March** 2025, **[10:00 - 16:00 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=HPC+Ready%3A+Developing+RSE+and+HPC+skills&iso=20250327T10&p1=1440&ah=6)**
+- 🌍 In person at the University of Edinburgh, Bayes Centre and Online via Zoom
+
+UNIVERSE-HPC is organising an event to mark the end of the three years of the UNIVERSE-HPC project. We'll have talks from members of the project as well as the wider RSE and HPC training community. This will be a great opportunity to meet and network with members of the community and find out more about the work going on in this area! :sparkles:
```
```suggestion
After three years, the UNIVERSE-HPC project is coming to a close and we will be celebrating this milestone with an event about training for RSEs, with an emphasis on HPC. Members of the wider RSE and HPC training communities will give talks on their work in this field and UNIVERSE-HPC members will share some of our achievements over the course of the project. This will be a great opportunity to meet and network with members of the community and find out more about the work going on in the area of RSE training! ✨ 
```

jcohen02 replied:
I don't think I can edit the suggested change above so just noting that in the edited version, the first line has a typo where it says "...we are want to....". You could perhaps say: "_After three years, the UNIVERSE-HPC project is coming to a close and we will be celebrating this milestone with an event..._".

Also, maybe expand HPC where it says with an emphasis on, i.e.: "_with an emphasis on High Performance Computing (HPC)._"?

eirini-zormpa commented on _pages/events/2025-03-27_project-end.md:
```diff
@@ -0,0 +1,62 @@
+- 10:30 - 11:15 Introduction and Opening Keynote
+- 11:15 - 11:30 Coffee break ☕
+- 11:30 - 12:15 Introduction to learning pathways and invited Ligthning talks
+- 12:15 - 13:00 Learning pathway activities (including reporting)
```
```suggestion
- 12:15 - 13:00 Break-out rooms: Learning pathway activities
```

eirini-zormpa commented on _pages/events/2025-03-27_project-end.md:
```diff
@@ -0,0 +1,62 @@
+- 10:00 - 10:30 Coffees and Meet and Greet ☕
+- 10:30 - 11:15 Introduction and Opening Keynote
+- 11:15 - 11:30 Coffee break ☕
+- 11:30 - 12:15 Introduction to learning pathways and invited Ligthning talks
```
```suggestion
- 11:30 - 12:15 Session on learning pathways
```

eirini-zormpa commented on _pages/events/2025-03-27_project-end.md:
```diff
@@ -0,0 +1,62 @@
+
+### Provisional agenda
+
+- 10:00 - 10:30 Coffees and Meet and Greet ☕
```
```suggestion
- 10:00 - 10:30 Meet and Greet ☕
```

eirini-zormpa commented on _pages/events/2025-03-27_project-end.md:
```diff
@@ -0,0 +1,62 @@
+- 13:00 - 14:00 Lunch break 🍱🥪
+- 14:00 - 14:45 Lightning talks
+- 14:45 - 15:00 Coffee break ☕
+- 15:00 -15:45 Closing Keynote
```
```suggestion
- 15:00 - 15:45 Closing Keynote
```

eirini-zormpa commented on _pages/events/2025-03-27_project-end.md:
```diff
@@ -0,0 +1,62 @@
+- 14:00 - 14:45 Lightning talks
+- 14:45 - 15:00 Coffee break ☕
+- 15:00 -15:45 Closing Keynote
+- 15:45 - 16:00 Closing remarks & evaluation survey
```
```suggestion
- 15:45 - 16:00 Closing remarks
```

eirini-zormpa reviewed (approved):
I've made some edits to the text - these are suggestions feel free to pick and choose what you want 😊 

Thank you for putting this together ✨

jcohen02 commented on _pages/events/2025-03-27_project-end.md:
```diff
@@ -0,0 +1,62 @@
+
+---
+
+**Join us for "HPC Ready: Developing RSE and HPC skills" on Thursday 27th March 2025 from 10:00 to 16:00 UTC**, which will take place **in person at the University of Edinburgh [Bayes Centre](https://bayes-centre.ed.ac.uk/), and online**.
```
Just a thought, the time makes it obvious but we could perhaps highlight that it's a one-day workshop since I feel that tends to engage people somewhat? e.g. 
```suggestion
**Join us for a one-day workshop - "HPC Ready: Developing RSE and HPC skills" - on Thursday 27th March 2025 from 10:00 to 16:00 (GMT/UTC)**, which will take place **in person at the University of Edinburgh [Bayes Centre](https://bayes-centre.ed.ac.uk/), and online**.
```

jcohen02 reviewed (commented):
Many thanks @eirini-zormpa, @SaranjeetKaur, this is looking great. I've added a few more comments and suggestions. Once you've finished with your edits, Eirini, feel free to go ahead and merge.

jcohen02 reviewed (commented):
Many thanks @eirini-zormpa, @SaranjeetKaur, this is looking great. I've added a few more comments and suggestions. Once you've finished with your edits, Eirini, feel free to go ahead and merge.

SaranjeetKaur commented:
@eirini-zormpa I had put "form to be updated" next to the registration form, because for some reason I could only see 5 questions in the form's preview instead of the 8 questions that I had added to it. I wasn't able to figure out why that was happening. Are you able to see 8 questions on the form?

eirini-zormpa commented:
@SaranjeetKaur oh interesting! I can only see 5 questions too - could you share the link to edit the survey?

eirini-zormpa commented:
oh right - it's the branching! the rest of the questions only appear once you select whether you're joining online or in person. However, the option to present a lightning talk only appears if you select the 'in-person' option. I think it's okay to allow that as an option to the people attending online too!

SaranjeetKaur commented:
> oh right - it's the branching! the rest of the questions only appear once you select whether you're joining online or in person. However, the option to present a lightning talk only appears if you select the 'in-person' option. I think it's okay to allow that as an option to the people attending online too!

@eirini-zormpa - this PR is already merged, but I am just going through it. So you say we allow online people to present lightning talks too. Were you able to modify the registration form to reflect the same? I can still see only 5 questions on the form and I don't remember doing any branching either.
